js/todolist/CheckItem.js

```diff
@@ -0,0 +1,22 @@
+import { API } from "../api.js";
+import { getAllTodoList, $todoApps, teamId } from "./showTodoList.js";
+
+export const initCheckTodoList = () => {
+  $todoApps.addEventListener("click", workCheck);
+};
+
+const workCheck = async ({ target }) => {
+  if (!target.classList.contains("toggle")) return;
+
+  const li = target.closest("li");
+  li.classList.toggle("completed");
+
+  if (target.checked) target.setAttribute("checked", "");
+  else target.removeAttribute("checked");
```

Comment:
```suggestion
target.toggleAttribute('checked');
```

```toggleAttribute()``` 메서드를 사용하면 코드가 간결해질거 같습니다!

- [toggleAttribute / MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/toggleAttribute)

Reply:
이전에도 사용했었는데, 왜 또 이렇게 작성해놨을까요 ㅠㅠ
review#3 에 반영하였습니다. 감사합니다!